patches/server/0176-Add-ArmorStand-Item-Meta.patch

```diff
@@ -13,11 +13,14 @@ starting point for future additions in this area.
 Fixes GH-559
 
 diff --git a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaArmorStand.java b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaArmorStand.java
-index 4017933f2244fca32cf9d39444f3a4f550e8af01..0c40a4a18cfc6f4ed6473a475f307f5c75ab56c5 100644
+index 4017933f2244fca32cf9d39444f3a4f550e8af01..d5a63727912f6775dd0eb3f4acbcbc553d1dc0b0 100644
 --- a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaArmorStand.java
 +++ b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaArmorStand.java
-@@ -8,9 +8,22 @@ import org.bukkit.Material;
+@@ -6,11 +6,25 @@ import net.minecraft.nbt.CompoundTag;
+ import net.minecraft.nbt.Tag;
+ import org.bukkit.Material;
  import org.bukkit.configuration.serialization.DelegateDeserialization;
++import org.checkerframework.checker.nullness.qual.Nullable; // Paper
```

Comment:
Idk if the nullable is needed, it's set as null so i'm guessing it can be implied. Just, import diffs are yucky

Reply:
Hm, what do you think about using the FQN?

Reply:
I think he means you just don't have to annotate the Type in the field as Nullable. Simply initializing it to null should be enough. We aren't picky about nullability annotations on the server, just api.